README.md

```diff
@@ -0,0 +1,20 @@
+# Digidecs
+Digidecs is the online form for committee- and board members to submit their receipts.
+
+# Prerequesites
+- [Yarn](https://classic.yarnpkg.com/en/)
+- [Vue](https://vuejs.org/)
+- [Rust](https://www.rust-lang.org/)
+
```

Comment:
Perhaps add that add least node 21 is used. With a recommendation to use nvm to manage it